tree-sitter-python/.ghci

```diff
@@ -5,3 +5,4 @@
 import TreeSitter.Unmarshal
 import Source.Span
 import Source.Range
+:set prompt "> "
```

Comment:
@aymannadeem is this OK? I like having a short ghci prompt without all the module names.

Reply:
yes, I wasn't sure if it was ok to be this prescriptive with other people's prompts. 😄

Reply:
@tclem: IMO this belongs in your user .ghci file, not in any particular project’s.

Reply:
E.g. here’s my ridiculous prompt: https://github.com/robrix/dotfiles/blob/master/home/.ghc/ghci.conf#L23-L34

Reply:
Ah, right on. Feel free to remove this the next time we're changing these files.